01-main/packages/freetube

```diff
@@ -0,0 +1,10 @@
+DEFVER=1
+ARCHS_SUPPORTED="amd64 arm64 armv7l"
```

Comment:
'armv7l' needs to be `armhf` (ie the relevant option from 
``` shell
dpkg-architecture -L |grep '^arm'
armhf
armel
arm64ilp32
armeb
arm
arm64
```
rather than the HOST_CPU


```suggestion
ARCHS_SUPPORTED="amd64 arm64 armhf"
```